browser/android/preferences/background_video_playback_tab_helper.cc

```diff
@@ -80,4 +84,74 @@ void BackgroundVideoPlaybackTabHelper::DidFinishNavigation(
   }
 }
 
+namespace chrome {
+namespace android {
+
+mojo::AssociatedRemote<script_injector::mojom::ScriptInjector> GetRemote(
+    content::RenderFrameHost* rfh) {
+  mojo::AssociatedRemote<script_injector::mojom::ScriptInjector>
+      script_injector_remote;
+  rfh->GetRemoteAssociatedInterfaces()->GetInterface(&script_injector_remote);
+  return script_injector_remote;
+}
+void JNI_BackgroundVideoPlaybackTabHelper_SendOrientationChangeEvent(
```

Comment:
this method name doesn't make sense to me, it's not sending an orientation change, it's toggling YT fullscreen

Reply:
also why are we putting this inside background_video_playback_tab_helper.cc?

Reply:
We are already doing background audio playback for youtube videos in this class. I'll change the class name.